P32-P35_P72-P75.s

```diff
@@ -107,7 +107,7 @@
 #			CALCULATES THE MANEUVER PARAMETERS.  SET AT THE START OF
 #			EACH RENDEZVOUS PRE-THRUSTING PROGRAM.
 #
-#			FINAL FLAG -- SELECTES FINAL PROGRAM DISPLAYS AFTER CREW HAS
+#			FINAL FLAG -- SELECTED FINAL PROGRAM DISPLAYS AFTER CREW HAS
```

Comment:
Both are wrong.

`SELECTS` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/0618.jpg)